### PR TITLE
Bump Symfony dependencies' versions

### PR DESCRIPTION
**Pull request by jakubtobiasz**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 2.0
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | n/a
| License         | MIT

### CODE REVIEW

**Review by Roshyo — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -90,51 +90,50 @@
...
+        "symfony/property-access": "^6.3.2",
+        "symfony/property-info": "^6.3.0",
+        "symfony/proxy-manager-bridge": "^6.3.0",
+        "symfony/routing": "^6.3.3",
+        "symfony/security-bundle": "^6.3.4",
+        "symfony/security-core": "^6.3.3",
+        "symfony/security-csrf": "^6.3.2",
+        "symfony/security-http": "^6.3.4",
+        "symfony/serializer": "^6.3.4",
         "symfony/service-contracts": "^2.5 || ^3.0",
```
Comment (diimpp) [inline composer.json:128]: It should be possible just by looking at packagist, something blocking it? (There are other instances of this dependency in other composer.json)

```patch
--- "symfony/service-contracts": "^2.5 || ^3.0",
+++ "symfony/service-contracts": "^3.3",
```

File: composer.json [hunk 2]
```diff
@@ -90,51 +90,50 @@
...
-        "symfony/property-access": "^5.4.21 || ^6.0",
-        "symfony/property-info": "^5.4.21 || ^6.0",
-        "symfony/proxy-manager-bridge": "^5.4.21 || ^6.0",
-        "symfony/routing": "^5.4.21 || ^6.0",
-        "symfony/security-bundle": "^5.4.21 || ^6.0",
-        "symfony/security-core": "^5.4.21 || ^6.0",
-        "symfony/security-csrf": "^5.4.21 || ^6.0",
-        "symfony/security-http": "^5.4.21 || ^6.0",
-        "symfony/serializer": "^5.4.21 || ^6.0",
+        "symfony/asset": "^6.3.0",
```
Comment (diimpp) [inline composer.json:93]: Just a note,

some of the `sylius-labs/polyfill-symfony-security` features are SF5 specific and can be scheduled to be dropped as well.

Other SyliusLabas polyfills should checked as well.

**Review by diimpp — APPROVED** [review]
Would be cool to keep track of "Drop Symfony5 support" somewhere, there are at least syliuslabs polyfills to check, twig2 is at the end of support by end of 2023, etc

**Comment by jakubtobiasz** [comment]
There'll be a separate task for reviewing all dependencies. I just needed to have only Symfony 6 😅. But you're right. I guess all polyfills will be dropped, as all other packages will be bumped.

File: src/Sylius/Component/Core/composer.json [hunk 3]
```diff
@@ -52,18 +52,18 @@
         "sylius/taxation": "^2.0",
         "sylius/taxonomy": "^2.0",
         "sylius/user": "^2.0",
-        "symfony/http-client": "^5.4.21 || ^6.0",
-        "symfony/http-foundation": "^5.4.21 || ^6.0",
+        "symfony/http-client": "^6.3.2",
+        "symfony/http-foundation": "^6.3.4",
         "symfony/mime": "^5.4.21 || ^6.0",
```
Comment (oallain) [inline src/Sylius/Component/Core/composer.json]:
```suggestion
        "symfony/mime": "^6.3.0",
```
IMO, we can bump `symfony/mime` too.

File: composer.json (hunk 1 above)
Comment (oallain) [reply composer.json:128]:
```
sylius/sylius                 v1.13.0-dev requires  symfony/service-contracts (^2.5 || ^3.0)         
doctrine/doctrine-bundle      2.10.2      requires  symfony/service-contracts (^1.1.1 || ^2.0 || ^3) 
sylius/theme-bundle           v2.3.0      requires  symfony/service-contracts (^1.1 || ^2.0 || ^3.0) 
symfony/cache                 v5.4.28     requires  symfony/service-contracts (^1.1|^2|^3)           
symfony/console               v5.4.28     requires  symfony/service-contracts (^1.1|^2|^3)           
symfony/dependency-injection  v5.4.28     requires  symfony/service-contracts (^1.1.6|^2)            
symfony/doctrine-bridge       v5.4.28     requires  symfony/service-contracts (^1.1|^2|^3)           
symfony/doctrine-messenger    v5.4.25     requires  symfony/service-contracts (^1.1|^2|^3)           
symfony/expression-language   v5.4.21     requires  symfony/service-contracts (^1.1|^2|^3)           
symfony/form                  v5.4.26     requires  symfony/service-contracts (^1.1|^2|^3)           
symfony/framework-bundle      v5.4.28     conflicts symfony/service-contracts (>=3.0)                
symfony/http-client           v5.4.26     requires  symfony/service-contracts (^1.0|^2|^3)           
symfony/mailer                v5.4.22     requires  symfony/service-contracts (^1.1|^2|^3)           
symfony/monolog-bridge        v5.4.22     requires  symfony/service-contracts (^1.1|^2|^3)           
symfony/security-core         v5.4.22     requires  symfony/service-contracts (^1.1.6|^2|^3)         
symfony/stopwatch             v5.4.21     requires  symfony/service-contracts (^1|^2|^3)             
symfony/twig-bundle           v5.4.27     conflicts symfony/service-contracts (>=3.0)                
symfony/webpack-encore-bundle v1.17.1     requires  symfony/service-contracts (^1.0 || ^2.0 || ^3.0) 

```

File: src/Sylius/Component/Core/composer.json (hunk 3 above)
Comment (jakubtobiasz) [reply src/Sylius/Component/Core/composer.json]: You can call yourself an eagle eye from now 😅

**Review by oallain — APPROVED** [review]
LGFM 👍🏼

**Review by loic425 — APPROVED** [review]

**Pull request merged**